docker/phpfpm/Dockerfile

```diff
@@ -0,0 +1,30 @@
+#
+# PHP Dependencies
+#
+FROM composer:1.7 as composer
+
+
+#
+# Application
+#
+FROM php:7.2-fpm AS base
```

Comment:
Perhaps we should test on a php 5.6 container image?

Reply:
I think that was an old commit this is already changed.